linkerd/distribute/src/cache.rs

```diff
@@ -1,51 +1,63 @@
-use super::{params, NewDistribute};
-use linkerd_stack::{layer, NewService, Param};
+use super::params;
+use linkerd_stack::{layer, ExtractParam, NewService};
 use parking_lot::Mutex;
 use std::{fmt::Debug, hash::Hash, sync::Arc};
 
-/// A [`NewService`] that produces [`NewDistribute`]s using a shared cache of
+/// A [`NewService`] that produces [`BackendCache`]s using a shared cache of
```

Comment:
this docs link is emits a warning since `BackendCache` is not publicly re-exported.